recipe/meta.yaml

```diff
@@ -18,16 +18,17 @@ build:
   entry_points:
     - wandb=wandb.cli.cli:cli
     - wb=wandb.cli.cli:cli
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
 
 requirements:
   host:
     - pip
     - wheel
     - python 
     - setuptools <64
```

Comment:
setuptools pinning can be relaxed, see https://github.com/wandb/wandb/blob/6c7524e215b6d597a4cd27ccd35b0ceaed240917/pyproject.toml#L2